src/main/scala/mesosphere/marathon/api/akkahttp/AppsDirectives.scala

```diff
@@ -0,0 +1,33 @@
+package mesosphere.marathon
+package api.akkahttp
```

Comment:
Hm, I'm not sure this class should go into this package. Isn't it v2 specific? Not a blocker though 🙂.

Reply:
I would be ok with it here for now. We'll probably have to rework package structure anyway.